docker/images/iron/Dockerfile

```diff
@@ -49,6 +49,8 @@ RUN apt-get update \
     python3-colcon-coveragepy-result \
     python3-colcon-lcov-result \
     python3-pip \
+    libboost-all-dev \
+    libopenvdb-dev \
```

Comment:
@pvela2017 nit: keep these sorted alphabetically.

Reply:
Changed in b6073eec0142379f885540be34e25213e0a9247d